src/pages/index.tsx

```diff
@@ -1,15 +1,17 @@
-import Head from 'next/head';
-import Link from 'next/link';
 import { Box, Button, Typography } from '@mui/material';
-import {useSession, signOut} from 'next-auth/react';
+import {getSession, useSession, signOut} from 'next-auth/react';
 import type { NextPageWithLayout } from '@common/types';
 import type { ReactElement } from 'react';
 import LeftNavLayout from '@common/layouts/LeftNavLayout';
-import classes from  '../styles/Home.module.css';
 
+const session = getSession();
```

Comment:
Oh, is this why it's working on the index.tsx page but not on the mainToolbar? Because on MainToolbar, I used `useSession`, but is it just that I should have used `getSession` instead? @samohansen

Reply:
the getSession method works, and returns a session object, but for some reason I can't access properties of the object like name and email, etc.. so I'm trying to get useSession() to work (unsuccessfully so far)